source/mainnet/smart-contracts/guides/integration-test-contract.rst

```diff
@@ -108,8 +111,10 @@ Deploying smart contract modules is a two-step process.
 First, you load the module with the function |module_load_v1|_, then you deploy it to the chain with the method |Chain_module_deploy_v1|_.
 Loading as a separate step allows you to reuse the loaded module across multiple tests for efficiency.
 
-The module to load should be a ``wasm`` module compiled with ``cargo concordium build``.
-For example, for ``cargo concordium build --embed-schema --out my_module.wasm.v1``, you write:
+The module to load should be a ``wasm`` module compiled with ``cargo concordium build`` or ``cargo concordium test --out path/to/wasm/module``.
+Using the test command is ideal, as that will both compile the module *and* run the tests.
+By compiling the module every time, you ensure that the tests run on the newest version of your code.
+For example, for ``cargo concordium test --embed-schema --out my_module.wasm.v1``, you write:
```

Comment:
I think we need to say somewhere that they need cargo-concordium version 2.9.0 at least for this to work.